### ISSUE TITLE
Avoid of using direct file names into configs

### ISSUE THREAD

**Issue by Grawl** [2gis/tars#53]
I am lurked into `tars/tasks/css/compile-css.js` and seen awful things:

```js
var scssFilesToConcatinate = [
        './markup/' + tarsConfig.fs.staticFolderName + '/scss/normalize.scss',
        './markup/' + tarsConfig.fs.staticFolderName + '/scss/libraries/**/*.scss',
        './markup/' + tarsConfig.fs.staticFolderName + '/scss/libraries/**/*.css',
        './markup/' + tarsConfig.fs.staticFolderName + '/scss/mixins.scss',
        './markup/' + tarsConfig.fs.staticFolderName + '/scss/sprites-scss/sprite_96.scss'
    ];
```

I don't know all the situation but I think that we can avoid such things with something like `usemin` ([`gulp-usemin`](https://www.npmjs.com/package/gulp-usemin) for example)

**Comment by artem-malko** [comment]
The main idea is controlling the order of file concatenating. It's not about minifactions, it's about ordering. Actually i can replace "'./markup/' + tarsConfig.fs.staticFolderName + '/scss/ to" by variable.

**Comment by Grawl** [comment]
Oh I got it. I control the order of injected with [`grunt-injector`](https://www.npmjs.com/package/grunt-injector) files by naming they using special characters `_` (or even `__`) and `!`:

```
_modules
  __fonts.sass
  __library.sass
  _settings.sass
  main.sass
sections
  footer.sass
  …
_dependencies.sass
style.sass
```

I think this way is better than writing file names right into build system.

**Comment by artem-malko** [comment]
@Grawl it's not obviously and hard to control. Actually, all files, which are included by their name, will have constant name. I use path pattern for other files, so, there is no problem with it.

**Comment by Grawl** [comment]
I'm just want to not touch build system if possible

**Comment by artem-malko** [comment]
@Grawl and you shouldn't touch it) You've got some places, there you can do anything (plugins, libraries and so on). And of course you can not use all files in static directory.

**Comment by Grawl** [comment]
I got it. But what if I want to change something? It will entail changing build system part where files is tacked with nails.

**Comment by artem-malko** [comment]
@Grawl you are free to do change anything. You've got uset-tasks and user user-watchers. Give an example of such changes.

**Comment by artem-malko** [comment]
@Grawl up!

**Comment by Grawl** [comment]
@artem-malko sorry for delay. Was hired and haven't time.

For example, I can want to replace ugly SCSS with clear and shiny tabbed Sass. So at least I have to rename `scss` to `stylesheets` (because now it will contain both TARS' `scss` and my `sass`).

**Comment by artem-malko** [comment]
@Grawl this problem will be solved by options in 1.3.0
#54
